### DISCUSSION TITLE
shelly rgbw2 not works for me, what do i wrong

### DISCUSSION THREAD

**Discussion by Mizayah** [RavenSystem/esp-homekit-devices#2270] [MEPLHAA Scripts]
Hello,

Just need an fast tip. I have flashed device to Raven system, and configure wifi after on device. Even after power off and on i cannot get to web interface of it anymore.  How to enter setup once more?

Also i have script like that
{"c":{"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[15],"b":[{"g":0}]},{"t":30,"g":[12],"b":[{"g":0}]},{"t":30,"g":[14],"b":[{"g":0}]},{"t":30,"g":[4],"b":[{"g":0}]}]}

homkit add 4 of lights, but after connection to 12v i still canot get lights up :(
may it be because my lights are not dimmable?

**Comment by Mizayah** [comment]
ok thanks to Haa manager i can enter setup mode, but still cant get it to work with led strip :(

**Comment by RavenSystem** [comment]
> Hello,
> 
> Just need an fast tip. I have flashed device to Raven system, and configure wifi after on device. Even after power off and on i cannot get to web interface of it anymore. How to enter setup once more?
> 
> Also i have script like that {"c":{"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[15],"b":[{"g":0}]},{"t":30,"g":[12],"b":[{"g":0}]},{"t":30,"g":[14],"b":[{"g":0}]},{"t":30,"g":[4],"b":[{"g":0}]}]}
> 
> homkit add 4 of lights, but after connection to 12v i still canot get lights up :( may it be because my lights are not dimmable?

Your script is wrong. Try this Shelly RGBW2 (4 different channels) script:

```json
{"c":{"io":[[[12,15],7],[[14,4],7,0,0,2],[[2],2],[[13],6,1],[[5],6,0,1]],"l":2,"f":20,"b":[[13,5]]},"a":[{"t":30,"g":[12]},{"t":30,"g":[15]},{"t":30,"g":[14]},{"t":30,"g":[4]}]}
```

**Reply by Mizayah** [reply]
YOU


ARE


MY 


H E R O



it works! :)
